### DISCUSSION TITLE
[OFF] Qual é a melhor opção para receber através de Payoneer?

### DISCUSSION THREAD

**Discussion by clmadrazo** [frontendbr/forum#1018] [Discussão]
Sou novo em Payoneer e queria saber a melhor opção para receber pagamentos do exterior por esta via. Alguém com experiencias que possa compartilhar?

**Comment by willycamargo** [comment]
Recebo do exterior pelo Payoneer faz alguns meses. O primeiro pagamento foi bem burocrático e tive que preencher vários formulários de Pessoa Jurídica e Pessoa Fisica. Precisei anexar alguns documentos também...

Na prática eu consegui receber mesmo 20 dias depois que o contratante fez o pagamento para a plataforma. Então se alguém pensa em usar o sistema eu recomendo ir já fazend o cadastro antecipadamente.

Os outros meses foram tranquilos, até 48h para o dinheiro cair na conta aqui do Brasil.

Ah, falando de fluxo, é o seguinte:

1. Gero um invoice usando [essa plataforma](https://app.freeinvoicebuilder.com)

1. Entro no Payoneer e gero uma cobrança para o email do contratante e coloco o invoice em anexo

1. Aguardo o pagamento (payoneer avisa por email quando for pago)

1. Entro no payoneer e peço o saque para a conta bancária cadastrada

1. Depois de recebido o dinheiro na minha conta, já em reais, gero a nota fiscal na plataforma da prefeitura como "serviço prestado para o exterior"

1. Cadastro a nota fiscal e a movimentação no Contabilivre (sistema de contabilidade que utilizo)

**Comment by clmadrazo** [comment]
@willycamargo sabe dizer as taxas cobradas nesse processo até chegar na tua conta bancária?

**Comment by willycamargo** [comment]
@clmadrazo primeiramente desculpe pela demora.

Pago 1% para Payoneer + taxa de cambio do dia (recebo em dolares e normalmente esta uns 10 centavos abaixo do dolar comercial)

**Comment by clmadrazo** [comment]
@willycamargo valeu cara, obrigado!

**Comment by lucianoviola** [comment]
> @clmadrazo primeiramente desculpe pela demora.
> 
> Pago 1% para Payoneer + taxa de cambio do dia (recebo em dolares e normalmente esta uns 10 centavos abaixo do dolar comercial)

essa taxa de cambio é a comercial? do google?

**Reply by renatomateussantos** [reply]
Isso é um absurdo. Hoje o dólar está 5.31, eles estão cotando em 5.18.

**Comment by willycamargo** [comment]
@lucianoviola

O câmbio que o Google mostra é o dólar comercial. O câmbio da Payoneer não é baseado nesse, o da Payoneer sempre fica uns 10 centavos ou mais de diferença (pra baixo)

**Comment by lcguida** [comment]
@clmadrazo @willycamargo Na verdade o payoneer coloca 2% em cima do dólar do google quando você saca para uma conta brasileira (https://www.payoneer.com/fees/).

No geral, eles te cobram pra receber e mais esses 2% na hora se sacar. Aí depende de como e de onde você vai receber. 

Pra receber via transferência bancária na Europa não é cobrado nada, por exemplo.

**Comment by clmadrazo** [comment]
@lcguida sim, é 2% mesmo da cotação do Google.

Para receber, se a empresa é cadastrada no Payoneer não cobram nada. Já se não é cadastrada cobram aquela porcentagem informada na tabela.

Até agora está sendo bem tranquilo e estão cumprindo com os prazos informados.

**Comment by nearutranslations** [comment]
@willycamargo 
O valor que devo colocar na nota fiscal é o valor que entra na sua conta jurídica ou o valor real que o cliente pagou (valor que entrou na conta jurídica + valor do serviço da Payonner)? 
Imagino que o correto seria a Payonner emitir nota fiscal sobre o valor do serviços dele e a gente colocar na nota fiscal o valor bruto (valor que entrou no banco mais o valor do serviço deles). Não sei se a Payonner tem escritório no Brasil.. se pode emitir nota no Brasil..

Obrigado

**Comment by lcguida** [comment]
> @willycamargo
> O valor que devo colocar na nota fiscal é o valor que entra na sua conta jurídica ou o valor real que o cliente pagou (valor que entrou na conta jurídica + valor do serviço da Payonner)?
> Imagino que o correto seria a Payonner emitir nota fiscal sobre o valor do serviços dele e a gente colocar na nota fiscal o valor bruto (valor que entrou no banco mais o valor do serviço deles). Não sei se a Payonner tem escritório no Brasil.. se pode emitir nota no Brasil..
> 
> Obrigado

Eles não tem escritório no Brasil. Eles fazem (como outros muitos fazem) uso de bancos intermediários/parceiros. Se você prestar atenção a TED que você recebe deles sempre vem de um outro banco.

**Comment by michaelfp** [comment]
Oi peseoal. Eu usei o global payment service para receber pela payoneer, alguém  ja usou? Ou vale a pena gerar um boleto para eles pagarem?

**Comment by lcguida** [comment]
@michaelfp Eu uso o payoneer. 

Se você estiver falando pro pessoal de fora do Brasil, eles não conseguem pagar boleto. Pra isso eles precisariam de uma conta no Brasil.

**Comment by michaelfp** [comment]
@lcguida  eu usei o Global payment Service que é uma conta que eles depositaram. Sabe quanto tempo demorar para cair na payoneer?

**Comment by lcguida** [comment]
Não depende do payoneer. Na verdade depende do banco do seu pagador. Assim que cai na conta do payoneer eles te enviam uma mensagem dizendo que você recebeu um pagamento. 

Aí você pode fazer o pedido de withdraw/trasnferencia para uma conta no Brasil. (Tem que cadastrar a conta antes). Quando eu faço esse pedido de manhã, costumo receber o dinheiro em reais no mesmo dia.

**Comment by michaelfp** [comment]
Eu to recebendo por semana. Eu posso fazer uma nota somente no final do mês com o que recebi para pagar os impostos?

**Comment by lcguida** [comment]
Se for PJ, você tem que fazer emitir a nota quando o dinheiro chega na sua conta PJ no Brasil.

Exemplo: Recebi no Payoneer no dia 05/07. Esperei um dias pra pedir a transferência. Fiz o pedido no dia 09/07, dinheiro cai na minha conta PJ no dia 10/07. Você emite a nota no dia 10/07.

**Comment by mayconfsbrito** [comment]
@lcguida acabei de fazer a minha primeira transferência para um banco brasileiro via Pioneer.
Você saberia me dizer se eu coloco os dados da empresa estrageira na nota fiscal? Tenho essa curiosidade pois empresa estrangeira não tem cadastro no Brasil (CNPJ e etc).

**Comment by lcguida** [comment]
O que o meu contador falou é fazer à nota sem CNPJ, e colocar as informações da empresa no corpo da nota. Essa emissão de nota é apenas pra contabilidade, sem necessidade de enviar p/ a empresa contratante.

--
Leandro Guida

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by iago-silva** [comment]
@lcguida top as informações que você compartilhou aqui.

Me diz uma coisa, se eu receber meu pagamento na payoneer no último dia do mês e eu só transferir para a conta BR no início do próximo mês, vai ter algum B.O na geração da nota fiscal e pagamento de impostos? Ou o tempo que o dinheiro ficou na Payoneer não importa pro leão?

Recebo como PJ.
